Bottom: fdb07e6e2b121ba4ee83390cd267ea9ed0f4b387
Top:    f94267064cfc106ece8a84863ae1118e0d6bebc1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 19:42:58 +0100

TMP HACK


---

diff --git a/src/misc.c b/src/misc.c
index 7c6f958..e001596 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -573,6 +573,12 @@ SetTextBoundingBox (FontTypePtr FontPtr, TextTypePtr Text)
    * and rotate box
    */
 
+  minx = MIL_TO_COORD (0);
+  miny = MIL_TO_COORD (0);
+  maxx = MIL_TO_COORD (500);
+  maxy = MIL_TO_COORD (500);
+
+
   if (TEST_FLAG (ONSOLDERFLAG, Text))
     {
       Text->BoundingBox.X1 = Text->X + minx;
